apps/backoffice-v2/src/pages/Entities/components/Cases/interfaces.ts

```diff
@@ -17,4 +17,5 @@ export interface ICasesProps extends DivComponent {
   onSortBy: ChangeEventHandler<HTMLSelectElement>;
   onSortDirToggle: () => void;
   search: string;
+  caseCounter: number;
```

Comment:
This property is in the context of cases, so the word `case` is redundant in this property name. The same as you don't call `user.user_email`, you call it `user.email`
```suggestion
  count: number;
```